collector/collector.cpp

```diff
@@ -154,17 +159,18 @@ void insertModule(const std::vector<std::string>& syscall_list) {
   module_args["exclude_selfns"] = "1";
   module_args["verbose"] = "0";
 
-  int fd = open(SysdigService::kModulePath, O_RDONLY);
-  if (fd < 0) {
-    CLOG(FATAL) << "Cannot open kernel module: " << SysdigService::kModulePath << ". Aborting...";
+  FDHandle fd = FDHandle(open(SysdigService::kModulePath, O_RDONLY));
+  if (fd.Invalid()) {
```

Comment:
I think this should be:
```suggestion
  if (!fd.valid()) {
```
`FDHandle::Invalid()` always returns -1, see: https://github.com/stackrox/collector/blob/130f7287b1f86d9c220dc37b06c30e2ecd897eca/collector/lib/FileSystem.h#L98
and: https://github.com/stackrox/collector/blob/130f7287b1f86d9c220dc37b06c30e2ecd897eca/collector/lib/FileSystem.h#L54

Reply:
ah yeah good spot, I'll fix this. Might have to go through that file in a follow up PR and make the naming a bit more consistent (feels like it should be `Valid()`)